### DISCUSSION TITLE
Is it possible to use a miller verb to remove an implicit header?

### DISCUSSION THREAD

**Discussion by pjfarleyiii** [johnkerl/miller#993] [Q&A] [answered]
I have a process that creates headerless CSV's from a gawk script, uses miller sort to sort them using the "--hi" flag and then appends them to a collection file using code like this:

```
gawk ...[gawk parameters]... | mlr --csv -hi sort -f 2 -n 5,6 >> final-collection-file.csv
```

I do NOT want the "--hi" header copied to the output.  Now, I know that can do it this way:

```
gawk ...[gawk parameters] | mlr --csv -hi sort -f 2 -n 5,6 | sed '1 d' >> final-collection-file.csv
```

But is there a way to use a miller capability to do undo the "--hi" addition of a header line for the sort process?

Peter

**Comment by aborruso** [comment, marked as answer]
Hi @pjfarleyiii , if you have

```
a,4
b,3
c,19
```

you can run

```
mlr --csv -N sort -f 2 input.txt
```

to have

```
c,19
b,3
a,4
```

`-N` is the same of `--implicit-csv-header` `--headerless-csv-output`

**Reply by pjfarleyiii** [reply]
Thank you, that was exactly what I needed to know.  I need to re-read that section of the docs a little more carefully.
